src/components/MDTemplates/AboutMeMD.tsx

```
@@ -0,0 +1,52 @@
+
+  return (
+    <AppCard
+      sx={{
+        mb: '2rem',
+      }}
+    >
+      <div
+        ref={contentRef}
+        className={styles.markdownBody}
+        data-theme={theme.palette.mode}
+        dangerouslySetInnerHTML={{ __html: md.render(preprocessedText) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)